modules/ravestate_ros2/ros2_properties.py

```diff
@@ -14,6 +14,7 @@
 ROS2_AVAILABLE = True
 try:
     import rclpy
+    import pyroboy
```

Comment:
pyroboy should be moved out of ros2_properties. 
Maybe some construction like ros2_properties.set_node_once() should be added to set the node from inside roboyio. It should check that it can only be called once to avoid overriding previous calls

other proposal:
ros2_properties.add_node() should be added so that other nodes get spun by ros2_properties